src/main/java/org/team1540/base/ChickenSubsystem.java

```diff
@@ -91,7 +91,7 @@ public double getCurrent() {
   @Override
   public void limitPower(double limit) {
     synchronized (powerLock) {
-      for (TalonSRX currentMotor : motors) {
+      for (ChickenTalon currentMotor : motors) {
         currentMotor.enableCurrentLimit(true);
         currentMotor.configContinuousCurrentLimit(Math.toIntExact(Math.round(limit / motors.size())), 0);
```

Comment:
If you're going to change it to `ChickenTalon`, may as well take advantage by removing the timeouts.